systemd/system/boot.automount

```diff
@@ -0,0 +1,16 @@
+#  This file is part of systemd.
```

Comment:
I presume you copied this from what the systemd generator creates but as-is it isn't really part of systemd any more. Is it worth preserving the header?